### ISSUE TITLE
XPT flag missing from README

### ISSUE THREAD

**Issue by ghost** [lennycolton/VFPC#161]
When opening https://github.com/VFPC/VFPC/issues/160, spotted that `XPT` is missing from the list of validation flags in the README.

**Comment by ghost** [comment]
It's not in the Training Dept guide to vFPC either.

**Comment by oldenough** [comment]
Since they didn't ask for any help when they did their training material it is probably from Lenny's and my efforts with the readme; missing there, missing everywhere.

**Comment by oldenough** [comment]
Not sure why you assigned this to Peter.  Did you see it other than an update to the README?

The plugin says:
```
vector<string> messageBuffer{ validize[0] }; // 0 = Callsign, 1 = SID, 2 = Destination, 3 = Exit Point, 4 = Route, 5 = Min/Max Flight Level, 6 = Even/Odd, 7 = Suffix, 8 = Restrictions, 9 = Warnings, 10 = Bans, 11 = Syntax, 12 = Passed/Failed
				...
else if (!messageBuffer.at(3).find("Failed")) {
			bufLog(flightPlan.GetCallsign() + string(": Fail - XPT"));
			return "XPT";
			```

**Comment by ghost** [comment]
No, only README, but maybe there are other undocumented errror codes, or some in the current list that aren't in the code any more.

**Comment by oldenough** [comment]
I got curious and went back to look.  ENG is in the readme list but not in the code. So, how DO we report this nowadays... I can't remember.

And we need to look at how BANS, Warnings and Alerts are described.  It could be significantly clearer.  We should probably get in the habit of warning training when we change the readme, since that seems to be what they are using.

Will the following work for the readme?
`XPT` - Exit point is not valid for route.

**Comment by ghost** [comment]
Or is it "Exit point not valid from this airport / SID"?

**Comment by oldenough** [comment]
Or even both! I just remembered that we are talking about the readme, so space isn't an issue.  😂

**Comment by oldenough** [comment]
XPT has been added to the Check Results section of README.md (between DST and RTE, which matches the evaluation order in getFails). ENG has been removed — see #175 for discussion of how engine-type restrictions should be handled going forward. Closing as resolved.
